### ISSUE TITLE
Issue to assign: Navbar missing on the AR Meditate page

### ISSUE THREAD

**Issue by ankitjt** [akshitagupta15june/Moksh#686]
## Before creating an issue, go through all open/closed issues to avoid getting your issue/suggestion marked as duplicate ##

**Issue and Steps to Reproduce:**
Navbar missing on the AR Meditate page.

**Screenshots:**

![image](https://github.com/akshitagupta15june/Moksh/assets/48979824/ee584dc8-33ef-4ebc-aa78-66324cdb4b4b)

**Expected:**
Page should have a functional and a responsive navbar.

It should be **in line with the navbar of the homepage** with regards to UI and functionality.

**Actual:**
NA

**Additional Details:**
After getting assigned contributor should submit their changes (Pull Request) with in a week, if it is not submitted with in a week the issue will be assigned to the next contributor in the comments section.

**Comment by khadar1020** [comment]
please assign this issue to me

**Comment by ankitjt** [comment]
No, refer the navbar on the homepage it should be same like that. Do a new fetch request to get the latest changes.
P.S. Navbar on the homepage is made using [Tailwind CSS](https://tailwindcss.com/), go through the documentation if you are not familiar with the framework.

**Comment by ankitjt** [comment]
@khadar1020 assigning the issue to you, pl submit your changes with in a week.

**Comment by ankitjt** [comment]
@khadar1020 when can you submit your changes?
